chapters/01-introduction.Rmd

```diff
@@ -15,6 +15,4 @@ Due to this increased robustness, KDE is particular useful in High-Energy Physic
 
```

Comment:
[8] In line 5: extend and make less complicated, shorter sentences. It's very dense currently and somewhat confusing: it seems like it's about multiple things at once. Explain when we can use parametric and when we can't. What we can do with non-parametric.

It seems all somewhat entangled, what exactly do you want to say in which paragraph?

The information is there, but it seems to be spread out